README.adoc

```diff
@@ -177,7 +177,7 @@ The following table illustrates a list of commands for various databases that ca
 
 |Oracle XE
 |`./docker_db.sh oracle`
-|`./gradlew test -Pdb=oracle_ci`
+|`./gradlew test -Pdb=oracle_free_ci`
```

Comment:
Hi, @loiclefevre @beikov, something appears to have got scrambled here. It seems to me that:

    ./docker_db.sh oracle
    ./gradlew test -Pdb=oracle_free_ci

Does not work, and that's not what you had in the previous PR.

What you had was:

```
|Oracle XE
| `./docker_db.sh oracle`
|`./gradlew test -Pdb=oracle_ci`

|Oracle FREE (23c+)
|`./docker_db.sh oracle_free`
|`./gradlew test -Pdb=oracle_free_ci`
```

And I strongly suspect that this is correct.

Reply:
`./docker_db.sh oracle` is an alias for `oracle_23` i.e. Oracle Free and `-Pdb=oracle_ci` was changed to work with Oracle Free. So yeah, the README is outdated. I'll fix that.

Reply:
Cool, thanks. 🙏